### ISSUE TITLE
Minimal README

### ISSUE THREAD

**Issue by libre-diane** [Tecuya/owgs#2]
I am currently wrapping my head into making this server works, it looks like a good base implementation to me to work on a fork project. 

However I kind of miss a clean and minimal README. I will eventually get the server started, but it's probably worth a few minutes to write a minimal doc on dependencies, setting up the database, how to start, pointing out relevant Django docs etc.

**Comment by Tecuya** [comment]
Yeah.  It's been years since I've touched this code.  A README would be good.  I have little time right now to do it.  If you have any questions I'd be happy to answer them.
